brave/brave-core
# Page content refine followup

Resolves https://github.com/brave/brave-browser/issues/40686
 - Fix potential crash when using custom models
 - Drop Leo prefix in local models updater


## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Setup custom model
2. Enable brave://flags/#brave-ai-chat-page-content-refine
3. Navigate to https://www.brianbondy.com/blog/187/cocodona-2024
4. Ask a question ex. "Who was the pacer?"
5. It should not crash and we should be able to get correct answer mentioning "Viktoria"
